### PR TITLE
IS-15196 Add support for teams endpoint

### PR DESCRIPTION
**Pull request by parhambrz**

### CODE REVIEW

File: templates/system.json [hunk 1]
```diff
@@ -35,6 +35,12 @@
       "payload_property": "data",
       "url": "{{ properties.datatype }}?{{ properties.param }}={{ entity.gid }}"
     },
+    "teams_list": {
```
Comment (parhambrz) [inline templates/system.json:38]: This is a copy of the tasks_list operation. Should we merge them with a more generic name?

Comment (mMaydew) [reply templates/system.json:38]: Yeah, if it's the same we can just update to a more generic name. I wasn't sure if any other datatypes would use this

**Review by mMaydew — COMMENTED** [review]
Could potentially use the same template as tasks. Iirc we can query tasks by workspace as well, will look into it later/Monday

**Review by mMaydew — APPROVED** [review]
LGTM! We can go through once all the datatypes are in and reduce the amount of templates etc

**Pull request merged**